core/src/main/resources/jenkins/model/Jenkins/login.jelly

```diff
@@ -72,74 +72,58 @@ THE SOFTWARE.
               <!-- login form -->
               <form name="login" action="${it.securityRealm.authenticationGatewayUrl}" method="post">
                 <j:if test="${it.securityRealm.allowsSignup()}">
-                  <div class="signupTag">
+                  <p class="signupTag">
```

Comment:
Would be nice if `.signuptag` had a `margin-top: 0` and a `margin-bottom: 2rem` - balances it out a little better 
<img width="365" alt="image" src="https://user-images.githubusercontent.com/43062514/163797532-f9d92467-39a0-4f25-ac75-242254440c6c.png">

Reply:
Same for the "If you have a Jenkins account..." and "A strong password is a long password th...." 

<img width="561" alt="image" src="https://user-images.githubusercontent.com/43062514/163797704-21e5ac24-1df6-4cf9-8b52-31c5163f9a2f.png">